lib/geoblacklight/download.rb

```diff
@@ -14,7 +14,7 @@ def file_name
     end
 
     def self.file_path
-      Settings.DOWNLOAD_PATH || "#{Rails.root}/tmp/cache/downloads"
+      Settings.DOWNLOAD_PATH || ENV['GEOBLACKLIGHT_DOWNLOAD_PATH'] || "#{Rails.root}/tmp/cache/downloads"
```

Comment:
@VivianChu Is it possible for you to use ENV vars to set your GeoBlacklight configurations?

So your local application might have something like this:

```yml
# config/settings.yml
DOWNLOAD_PATH: <%= ENV['GEOBLACKLIGHT_DOWNLOAD_PATH'] %>
```

My only concern with changing this in the GeoBlacklight code is that all of our configurations are consolidated using the config gem and this might start to expand that.

Reply:
@mejackreed - Thanks for the suggestion. It works fine for me.  I'll close this PR.

Reply:
Oh thats great to hear! 

Thanks for the contribution and raising the issue. Feel free to raise any other config issues you might run into.

Welcome 👍